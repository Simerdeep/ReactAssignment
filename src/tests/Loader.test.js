import React from "react";
import { mount } from "enzyme";
import Loader from "../components/Loader"

describe("Loader test cases",() => {

    const loaderViewWrapper = mount(<Loader/>);

    it("should render the snapshot for loader view", () => {
        expect(loaderViewWrapper).toMatchSnapshot();
    });

})
