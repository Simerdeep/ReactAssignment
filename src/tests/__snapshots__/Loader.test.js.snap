// Jest Snapshot v1, https://goo.gl/fbAQLP

exports["Loader test cases should render the snapshot for loader view 1"] = "ReactWrapper {}";
